### ISSUE TITLE
Visual Editor visual issues with MW 1.35

### ISSUE THREAD

**Issue by dhowland** [jthingelstad/foreground#381]
### Setup

- Browsers and versions: Firefox 83.0
- MediaWiki version: 1.35.0
- Database version: 5.7.32
- PHP version: 7.4.12
- Foreground version: git pull develop
- Extension version (if connected to the issue):

### Issue

Fresh mediawiki install.  Edit page with VE.

Screenshot https://i.imgur.com/NhZhGIP.png

**Comment by Hutchy68** [comment]
No, it was the previous fixes needed to make VE work with the skin. Seems as if VE now has targeted their CSS properly so the fixes are not needed any longer. In short, they are conflicting.

I will push a fix as soon as @jthingelstad looks at CircleCI and turn on its integration or let me know he's wants to stick with Travis-CI. I have updated my fork to stop using Travis-CI, start using CircleCI, and other improvements to testing. Travis-CI has basically become un-useable for open source project unless you have a paid account. This all started around Nov 2, 2020.

**Comment by dhowland** [comment]
Ok thanks.

Would it be appropriate to have some kind of light box around the editing window?  I ask because the nature of this skin makes footers and any other wiki stuff indistinguishable from article content.

**Comment by jthingelstad** [comment]
> I will push a fix as soon as @jthingelstad looks at CircleCI and turn on its integration

Done! 😄

**Comment by Hutchy68** [comment]
Ok, the fix is pushed to the `develop` branch for the VE toolbar. 

>Would it be appropriate to have some kind of light box around the editing window? I ask because the nature of this skin makes footers and any other wiki stuff indistinguishable from article content.

Let's all look at that first. You can try to add this to a MediaWiki message, create the page `MediaWiki:Foreground.css` then cut and paste the following into that new system message.

```
.ve-init-mw-desktopArticleTarget-originalContent {
    border-bottom: 2px solid #c8ccd1;
    padding: 15px 0;
}
```

*Note - I had a few issues with MW 1.35 and system message. 1.35 is still buggy IMHO. The page was in a loop on saving until I deleted the last bracket and then manually typed it in. Then I was allowed to save the page, MediaWiki:Foreground.css. So much for the easy cut and paste. I am finding changing the skin preference buggy too. Multiple tries until it takes. That's all MW so I guess I need to see if there are bugs reported about these issues yet.
